### PR TITLE
Bump Travis versions

### PR DESCRIPTION
**Pull request by HaraldNordgren**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ sudo: false
 go:
   - "1.9"
```
Comment (nywilken) [inline .travis.yml]: When updating supported versions on Travis we tend to drop support for the second oldest version (i.e 1.9). 
```suggestion
```

**Comment by HaraldNordgren** [comment]
@nywilken I would rather test a wider array of Go versions to maintain backward compatibility as long as it is viable. Could you explain you reasoning a bit further? 🙂

**Comment by kytrinyx** [comment]
@HaraldNordgren This only affects people contributing to the package, not people using the CLI. We find that overall Go developers upgrade within the year, and if they don't we feel it's ok if they get slower feedback by seeing failing tests in CI.

**Comment by HaraldNordgren** [comment]
@nywilken @kytrinyx Understood. I made sure to change `CONTRIBUTING.md` as well since the one change doesn't make much sense without the other, and vice versa.

**Comment by nywilken** [comment]
@HaraldNordgren sorry for the slow response on this. Thank you for making the updates.

**Pull request merged**